src/game/frontend/submenus/Teleport.cpp

```diff
@@ -1,14 +1,200 @@
 #include "Teleport.hpp"
+
+#include "game/features/Features.hpp"
 #include "game/frontend/items/Items.hpp"
+#include "game/services/custom_teleport_service.hpp"
+#include "util/Math.hpp"
+#include "util/Teleport.hpp"
+#include "core/frontend/Notifications.hpp"
+#include "game/backend/FiberPool.hpp"
 
 namespace YimMenu::Submenus
 {
+	Telelocation GetLocationPlayerIsClosestTo()
+	{
+		if (!Pointers.GetLocalPed)
+			return {};
+
+		float distance = 500;
+		Telelocation closestLocation{};
+		//saved_locations_filtered_list can be used to get a joint list of all categories when the filter is empty.
+		for (auto& loc : g_CustomTeleportService.SavedLocationsFilteredList())
+		{
+			float newDistance = Math::DistanceBetweenVectors(Self::Pos, {loc.x, loc.y, loc.z});
+
+			if (newDistance < distance)
+				closestLocation = loc, distance = newDistance;
```

Comment:
Whats up with the comma? Did i miss something?

Reply:
Thats valid syntax.